packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -44,7 +43,13 @@ export enum AttachState {
     Detached = "Detached"
 }
 
-export { ConnectionState }
+// @public (undocumented)
+export enum ConnectionState {
```

Comment:
i'd much rather we move this to a const object before making it public

Reply:
It's already public and exported by this package. These changes just make it more obvious that it's here.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6844

Reply:
I added some basic details to it, but feel free to add your thoughts.